_rest_routes.inc.php

```diff
@@ -33,6 +34,7 @@
 use OpenEMR\RestControllers\ImmunizationRestController;
 use OpenEMR\RestControllers\InsuranceRestController;
 use OpenEMR\RestControllers\MessageRestController;
+use OpenEMR\RestControllers\CodesRestController;
```

Comment:
hi @amolicas , pretty trivial but keeping these sorted alphabetically, thank you

Reply:
Hi @stephenwaite, sorry this is my first contribution, do you mean to sort this way?

use OpenEMR\RestControllers\CodesRestController;
use OpenEMR\RestControllers\DocumentRestController;
use OpenEMR\RestControllers\ImmunizationRestController;
use OpenEMR\RestControllers\InsuranceRestController;
use OpenEMR\RestControllers\MessageRestController;

Reply:
no prob, yes, looks perfect, and welcome aboard!

Reply:
Ok, I will commit this way, thanks!